### ISSUE TITLE
short case, false positive

### ISSUE THREAD

**Issue by larshp** [larshp/abaplint#395]
should allow the following,
```abap
        CASE ls_token-str.
          WHEN '+' OR '-' OR '*' OR '/' OR 'MOD' OR 'DIV' OR 'BIT-AND' OR 'BIT-OR' OR '&&'.
```
